source/_integrations/iperf3.markdown

```diff
@@ -121,4 +121,4 @@ Example Service data:
 
 ## Notes
 
-- When running on Raspberry Pi, just note that the maximum speed is limited by its 100 Mbit/s LAN adapter.
+- When running on Raspberry Pi < 3B+, just note that the maximum speed is limited by its 100 Mbit/s LAN adapter.
```

Comment:
Let's remove this whole line/section from the file.

On the Pi3 the LAN speed is actually limited by the USB bus, so while its negates on gigabit, it actually never reach that.

This piece is probably from the RPi 1&2 age.

Reply:
good idea. There will be too many exceptions to count them all.